jenkins/scripts/integration_test.sh

```diff
@@ -54,13 +54,10 @@ KUBERNETES_VERSION_UPGRADE_FROM="${KUBERNETES_VERSION_UPGRADE_FROM:-}"
 KUBERNETES_VERSION_UPGRADE_TO="${KUBERNETES_VERSION_UPGRADE_TO:-}"
 KUBECTL_SHA256="${KUBECTL_SHA256:-}"
 
-if [[ "${IRONIC_INSTALL_TYPE}" == "source" ]]; then
+# Build ironic locally when running tests on ironic-image repo  
+if [[ "${REPO_NAME}" == "ironic-image" ]]; then
```

Comment:
We do not want to always build from source. RPM installation is the default method, source building is special case.

We need to have source build triggerable via a keyword for those special cases, and default should stay RPM based install. It will still build the ironic container with the changes via `tested_repo`, as we can see from the e2e keyword tested yesterday, since https://github.com/metal3-io/ironic-image/pull/495 failed as it depends on https://github.com/metal3-io/ironic-image/pull/493 which succeeded.

/hold